File: app/src/main/java/com/hikvision/skinpeeler/app/MyApplication.java
package com.hikvision.skinpeeler.app;

import android.app.Application;

import com.hikvision.skinlibrary.SkinManager;

/**
 * 〈一句话功能简述〉
 * 〈功能详细描述〉
 *
 * @author hanpei
 * @version 1.0, 2019/9/6
 * @since 产品模块版本
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        SkinManager.init(this);
    }
}

File: app/src/main/java/com/hikvision/skinpeeler/app/MyApplication.kt
package com.hikvision.skinpeeler.app

import android.app.Application
import com.hikvision.skinlibrary.SkinManager
import com.hikvision.skinpeeler.utils.SharedPreferencesUtil
import com.hikvision.skinpeeler.utils.threadpool.AppExecutors

/**
 *
 * @author  Tyhj
 * @date    2019-12-12
 *
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        SkinManager.init(this)
        SharedPreferencesUtil.init(this)
        AppExecutors.getInstance()
    }
}